### PR TITLE
The user should be able to update his profile

### PR DESCRIPTION
**Pull request by Faustin250**
>Pass auth key into the header and proceed with the following

>Pass auth key into the header and proceed with the following

>Pass auth key into the header and proceed with the following

### CODE REVIEW

File: src/controllers/usersController.js [hunk 1]
```diff
@@ -197,4 +197,49 @@ export default class user {
...
+
+
+  static async updateProfile(req, res) {
+    try {
+      const { id } = req.params;
+      const { firstName } = req.body;
+      const { lastName } = req.body;
+      const { email } = req.body;
+      const { preferedLanguage } = req.body;
+      const { officeAddress } = req.body;
```
Comment (nkalyesubula) [inline src/controllers/usersController.js]: use code destructuring here

**Review by nkalyesubula — COMMENTED** [review]
Use code destructuring where necessary

File: src/app.js [hunk 2]
```diff
@@ -7,7 +7,10 @@ import swaggerDocument from './swagger/index';
 import router from './routes/index';
 
 const app = express();
-
+const fileupload = require('express-fileupload');
```
Comment (muhireheir) [inline src/app.js:10]: Do we need this?

**Review by nishimweelysee — APPROVED** [review]
This works Well

**Review by muhireheir — APPROVED** [review]
Approved!

File: src/app.js (hunk 2 above)
Comment (Faustin250) [reply src/app.js:10]: Yes. we need it. That package makes easier during file upload

**Comment by nkalyesubula** [comment]
LGTM

**Pull request merged**